ts/sagas/wallet/pagopaApis.ts

```diff
@@ -482,7 +484,7 @@ export function* paymentCheckRequestHandler(
     } else {
       yield put(paymentCheck.failure(undefined));
     }
-  } catch {
+  } catch (error) {
```

Comment:
`(error)` seems unused here. Maybe `paymentCheck.failure` payload type definition could be expanded to include error description

Reply:
updated within 7fd3edd1095bee221db15492d60b85eb6a2cfcb6